### ISSUE TITLE
The chat button obscures the purchase total information

### ISSUE THREAD

**Issue by jimkring** [canonical/ubuntu.com#13011]
![image](https://github.com/canonical/ubuntu.com/assets/381432/805c3326-0934-4106-a41d-7379cd4f1a1a)

---
*Reported from: https://ubuntu.com/pro/subscribe*

**Comment by anthonydillon** [comment]
@leiaru-c would you be able to suggest an improvement here?

**Comment by leiaru-c** [comment]
Indeed, live chat should not be enabled on the shop pages. It needs to be disabled on the shop and checkout :)

**Comment by caldav** [comment]
I'm a bit concerned, because it seems obvious to have this as a "I need help" button. It seems we can dynamically move its position based on the page. If we were to do that, where could we put it?

**Comment by immortalcodes** [comment]
Hey @jimkring ,
Could you please check if this issue still exists?

**Comment by minkyngkm** [comment]
Closing this issue as it's not relevant anymore. Live chat doesn't exist on https://ubuntu.com/pro/subscribe anymore.
